app.py

```diff
@@ -1,21 +1,20 @@
 # Import libraries
 import numpy as np
 import pandas as pd
-from sklearn import grid_search
-from sklearn.cross_validation import train_test_split
 
 from lib.data.wrangler import readData
 from lib.helpers import runTests, writeToCsv
 
-X_test, \
-    X_train, \
+X_train, \
```

Comment:
That's no good. I had to reverse on my end to get it to run... Is the pickling process deterministic?